scripts/README.md

```diff
@@ -33,10 +33,15 @@ If you belong to the [CI-A Github team](https://github.com/orgs/mozilla/teams/ci
 Open [this secret](https://community-tc.services.mozilla.com/secrets/project%2Fcia%2Fgarbage%2Ffoo) and try to see it. If you can see the
 secret you can then create your own.
 
-### Retrieving secrets locally
+## Run task on local machine 
+
+
+    set TASKCLUSTER_ROOT_URL=https://community-tc.services.mozilla.com
```

Comment:
Why do you need to set this?

Reply:
this is required for the taskcluster library to know where to go